MODEL-GUIDE.md

```diff
@@ -314,6 +315,95 @@ The build script will enforce the following keyword conventions:
     * _"Number of times `session_state` changed from `up` to `down`"_ instead of
     * _"Number of times the session went from Up to Down state"_
 
+### Field-UID
```

Comment:
Seems like this whole section would be better placed in the Keyword Extension section.

Reply:
done